site_preference_builder/customSitePrefBuilder.js

```diff
@@ -286,3 +298,59 @@ exports.addCustomGroupFromPlan = (file, plans) => {
 
   return file;
 };
+
+exports.addRefundCustomAttributes = (file) => {
+  file.metadata['type-extension'][0]['custom-attribute-definitions'][0]['attribute-definition'].push(
+    buildCustomSitePrefObject({
+      id: 'almaRefundedAmount',
+      name: 'ALMA Refunded Amount',
```

Comment:
Could you add theses in a refund key in the message.js file ? we might want to translate it someday, for now you can put English everywhere for now